src/models/externals/IExternalRoutePathNode.ts

```diff
@@ -0,0 +1,15 @@
+import IExternalRoutePath from './IExternalRoutePath';
+
+export default interface IExternalRoutePathNode {
```

Comment:
I find it a bad idea to have `IExternalRoutePathNode` and `IExternalNode` as different things

Reply:
Outdated, we now only have IExternalNode -> resolving